<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmWK6500PUi</class>
 <widget class="QDialog" name="frmWK6500PUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Agency FB</family>
    <pointsize>7</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Wayne Kerr 65120P Simulator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#frmWK6500PUi{
	border: 2px solid gray;
	background-color: #9595FF
}</string>
  </property>
  <widget class="QWidget" name="wgtMenuBackground" native="true">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>2</y>
     <width>149</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(185, 185, 185);</string>
   </property>
   <widget class="QWidget" name="wgdMenu" native="true">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>5</y>
      <width>144</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#wgdMenu
{
	background-color: #9595FF;
	border: 1px solid black;
}</string>
    </property>
    <widget class="QPushButton" name="btnFileMenu">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>46</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>46</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>22222</width>
       <height>2000</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#btnFileMenu:hover
{
	color:white
}</string>
     </property>
     <property name="text">
      <string>&amp;File</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btnModeMenu">
     <property name="geometry">
      <rect>
       <x>44</x>
       <y>0</y>
       <width>46</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>46</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>22222</width>
       <height>2000</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#btnModeMenu:hover
{
	color:white
}</string>
     </property>
     <property name="text">
      <string>&amp;Mode</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btnDisplayMenu">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>0</y>
       <width>52</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>52</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>2222</width>
       <height>2000</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#btnDisplayMenu:hover
{
	color:white
}</string>
     </property>
     <property name="text">
      <string>&amp;Display</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>34</y>
     <width>636</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(193, 193, 193);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btnShowScale">
   <property name="geometry">
    <rect>
     <x>152</x>
     <y>3</y>
     <width>35</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>6</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Show
Scale</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSaveNorm2">
   <property name="geometry">
    <rect>
     <x>224</x>
     <y>3</y>
     <width>35</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>6</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Save
Nom2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSaveNorm1">
   <property name="geometry">
    <rect>
     <x>187</x>
     <y>3</y>
     <width>35</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>6</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Save
Nom1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnResSearch">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>3</y>
     <width>40</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>6</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Res
Search</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnQuit">
   <property name="geometry">
    <rect>
     <x>597</x>
     <y>3</y>
     <width>40</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
  <widget class="QFrame" name="frmMain">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>35</y>
     <width>636</width>
     <height>421</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame#frmMain
{
	background-color:#E0E0FF;
	border: 2px solid #c1c1c1
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="wgtLeftPannel" native="true">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>130</width>
      <height>411</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#wgtLeftPannel
{
	background-color:#9595FF;
	 
	border:2px solid #c1c1c1;
}</string>
    </property>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>5</y>
       <width>121</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="title">
      <string>Measurement</string>
     </property>
     <widget class="QPushButton" name="btnItem1">
      <property name="geometry">
       <rect>
        <x>65</x>
        <y>16</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="text">
       <string>Z</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnItem2">
      <property name="geometry">
       <rect>
        <x>65</x>
        <y>48</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="text">
       <string>θ</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnEqucct">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>80</y>
        <width>111</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="text">
       <string>Series</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>25</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="text">
       <string>Term 1</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>55</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="text">
       <string>Term 2</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>120</y>
       <width>121</width>
       <height>55</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="title">
      <string>Speed</string>
     </property>
     <widget class="QPushButton" name="btnSpeed">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>20</y>
        <width>111</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="text">
       <string>Maximum</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="wgtRightPannel" native="true">
    <property name="geometry">
     <rect>
      <x>501</x>
      <y>5</y>
      <width>130</width>
      <height>411</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#wgtRightPannel
{
	background-color:#9595FF;
	 
	border:2px solid #c1c1c1;
}</string>
    </property>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>5</y>
       <width>121</width>
       <height>121</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="title">
      <string>Driver</string>
     </property>
     <widget class="QPushButton" name="btnLevel">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>85</y>
        <width>111</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="text">
       <string>1.0000V</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>15</y>
        <width>59</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="text">
       <string>Frequency</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="text">
       <string>Level</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnFrequency">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>35</y>
        <width>111</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="text">
       <string>1.0000kHz</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>130</y>
       <width>121</width>
       <height>55</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="title">
      <string>Range</string>
     </property>
     <widget class="QPushButton" name="btnRange">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>20</y>
        <width>111</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="text">
       <string>Auto</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>190</y>
       <width>121</width>
       <height>131</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="title">
      <string>Bias</string>
     </property>
     <widget class="QPushButton" name="btnBiasStatus">
      <property name="geometry">
       <rect>
        <x>74</x>
        <y>90</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="text">
       <string>OFF</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnBiasType">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>90</y>
        <width>65</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="text">
       <string>Current</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnBiasValue">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>40</y>
        <width>111</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="text">
       <string>0.0000A</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="text">
       <string>Value:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>28</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="text">
       <string>Type</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>75</x>
        <y>70</y>
        <width>35</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="text">
       <string>Status</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="btnRepeat">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>330</y>
       <width>111</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="text">
      <string>Single</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnTrig">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>360</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="text">
      <string>Trigger</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>15</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Meter Mode</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="txtItem1Res">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>80</y>
      <width>351</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>38</pointsize>
     </font>
    </property>
    <property name="text">
     <string>189.694kΩ</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="txtItem2Res">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>160</y>
      <width>351</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>38</pointsize>
     </font>
    </property>
    <property name="text">
     <string>-89.8118 °</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="wgdBiasON" native="true">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="pixmap">
      <pixmap resource="../Res/WKRes.qrc">:/BiasOn.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>45</x>
       <y>10</y>
       <width>271</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Tahoma</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 0, 0);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Bias on - Safety Hazard</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>10</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="pixmap">
      <pixmap resource="../Res/WKRes.qrc">:/BiasOn.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="txtMessage">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>454</y>
     <width>636</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="wgdLanRemote" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <widget class="QLabel" name="lblLanRemote">
    <property name="geometry">
     <rect>
      <x>535</x>
      <y>3</y>
      <width>101</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 0, 255);</string>
    </property>
    <property name="text">
     <string>Lan Remote</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <zorder>wgdLanRemote</zorder>
  <zorder>wgtMenuBackground</zorder>
  <zorder>line</zorder>
  <zorder>btnShowScale</zorder>
  <zorder>btnSaveNorm2</zorder>
  <zorder>btnSaveNorm1</zorder>
  <zorder>btnResSearch</zorder>
  <zorder>btnQuit</zorder>
  <zorder>frmMain</zorder>
  <zorder>txtMessage</zorder>
 </widget>
 <tabstops>
  <tabstop>btnFileMenu</tabstop>
  <tabstop>btnModeMenu</tabstop>
  <tabstop>btnDisplayMenu</tabstop>
  <tabstop>btnShowScale</tabstop>
  <tabstop>btnSaveNorm1</tabstop>
  <tabstop>btnSaveNorm2</tabstop>
  <tabstop>btnResSearch</tabstop>
  <tabstop>btnQuit</tabstop>
  <tabstop>btnItem1</tabstop>
  <tabstop>btnItem2</tabstop>
  <tabstop>btnEqucct</tabstop>
  <tabstop>btnSpeed</tabstop>
  <tabstop>btnFrequency</tabstop>
  <tabstop>btnLevel</tabstop>
  <tabstop>btnRange</tabstop>
  <tabstop>btnBiasValue</tabstop>
  <tabstop>btnBiasType</tabstop>
  <tabstop>btnBiasStatus</tabstop>
  <tabstop>btnRepeat</tabstop>
  <tabstop>btnTrig</tabstop>
 </tabstops>
 <resources>
  <include location="../Res/WKRes.qrc"/>
 </resources>
 <connections/>
</ui>
